mpp/mpp.F90

```diff
@@ -673,8 +682,19 @@ module mpp_mod
 #endif
   end interface
 
-  !> @brief Gather information onto root pe
+  !> @brief Gather data sent from pelist onto the root pe
+  !! Wrapper for MPI_gather, can be used with and without indices
   !> @ingroup mpp_mod
+  !!
+  !> @param sbuf MPP_TYPE_ data buffer to send
```

Comment:
Is `\param` supposed to be for input arguments for a function/routine?  Are these the input arguments for these interfaces?

Reply:
Yeah thats the required input args, it shows up as arguments for the interfaces too.